eva-release/src/main/java/uk/ac/ebi/eva/release/repositories/ReleaseStatsPerSpeciesRepository.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2020 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package uk.ac.ebi.eva.release.repositories;
+
+import org.springframework.data.repository.CrudRepository;
+import org.springframework.stereotype.Repository;
+
+import uk.ac.ebi.eva.release.models.ReleaseStartsPerSpeciesPK;
+import uk.ac.ebi.eva.release.models.ReleaseStatsPerSpecies;
+
+@Repository
+public interface ReleaseStatsPerSpeciesRepository extends CrudRepository<ReleaseStatsPerSpecies,
+        ReleaseStartsPerSpeciesPK> {
+
+    Iterable<ReleaseStatsPerSpecies> findAllByReleaseVersion(int releaseVersion);
+
+    //All release data excluding rows with only unmapped data
+    Iterable<ReleaseStatsPerSpecies> findByCurrentRsNotAndMultiMappedRsNotAndMergedRsNotAndDeprecatedRsNotAndMergedDeprecatedRsNotAndUnmappedRsGreaterThan(
```

Comment:
This method name looks unwieldy but I guess this is to satisfy the naming convention for Spring... Is there any workaround to shorten the method name?...

Reply:
yes, I don't like a lot this approach and there are ways to avoid using those long method names but didn't want to get into that in this first iteration of the API. I agree it should be changed